ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-answer/activityAnswer.component.ts

```diff
@@ -49,7 +49,12 @@ import { BlockVisibility } from '../../../models/activity/blockVisibility';
                                    [block]="block"
                                    [readonly]="readonly"
                                    (valueChanged)="onChange($event)">
-    </ddp-activity-agreement-answer>`
+    </ddp-activity-agreement-answer>
+    <span *ngIf="block.additionalInfoFooter && block.shown"
```

Comment:
I noticed that every block/component into the template has ` && block.shown` in their condition.
What do you think is it not worth extracting it and moving into the component level, like: 
```
 <ng-container *ngIf="block.shown">
      <ddp-activity-boolean-answer *ngIf="isBooleanQuestion(block)"></>
      <ddp-activity-text-answer *ngIf="isTextQuestion(block)"></>

```      etc...